functions-app/report-psychometrician/service/detections/get-check-started-date.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+
+const moment = require('moment')
+const R = require('ramda')
+
+function getCheckStartedDateFromAuditLog (payload) {
+  if (typeof payload !== 'object') return undefined
+  const audits = R.prop('audit', payload)
+  if (!audits) return undefined
+  const checkStartedEvent = R.find(R.propEq('type', 'CheckStarted'), audits)
+  if (!checkStartedEvent) return undefined
+  const ts = R.prop('clientTimestamp', checkStartedEvent)
+  if (!ts) return undefined
```

Comment:
is specifically undefined required here as a returned value?

Reply:
What would you suggest?

Reply:
Wouldn't plain return be sufficient?I'd expect the caller to handle it properly.

Reply:
Adjusted as you suggest :)